core/src/display_object/interactive.rs

```diff
@@ -475,6 +475,7 @@ pub trait TInteractiveObject<'gc>:
 pub enum Avm2MousePick<'gc> {
     Hit(InteractiveObject<'gc>),
     PropagateToParent,
+    NotAvm2,
     Miss,
```

Comment:
It looks like this PR creates instances of this variant, but never explicitly tests against it. Can we just use `Miss` instead, or do you anticipate needing to know that a mouse pick failed specifically due to an avm1 target?

Reply:
`Miss` and `NotAvm2` _should_ be the same thing, I was just trying to figure out how to differentiate between the two.